src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_kds.c

```diff
@@ -224,10 +224,18 @@ static int xocl_command_ioctl(struct xocl_dev *xdev, void *data,
 	/* TODO: one ecmd to one xcmd now. Maybe we will need
 	 * one ecmd to multiple xcmds
 	 */
-	if (ecmd->opcode == ERT_CONFIGURE) {
+	switch (ecmd->opcode) {
+	case ERT_CONFIGURE:
 		cfg_ecmd2xcmd(to_cfg_pkg(ecmd), xcmd);
-	} else if (ecmd->opcode == ERT_START_CU)
```

Comment:
Thank you for the mini-refactoring you are doing here.  Makes the code easier to read.